src/main/webapp/dev/js/test/CommonTestFunctions.es6

```diff
@@ -33,20 +33,21 @@ function clearBootboxModalStub() {
     $('#test-bootbox-modal-stub').html('');
 }
 
+const bootboxStub = {
```

Comment:
Haha alright. So something like `jQueryObjectStubForBootbox` would be better?

PS: 7 more comments and we hit 100 :p